lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -109,6 +127,16 @@ def get(self, request, *args, **kwargs):
         is_enrolled = enrollment and enrollment.is_active
         is_staff = has_access(request.user, 'staff', course_key)
 
+        course_outline_root_block = get_course_outline_block_tree(request, course_key_string, request.user)
+        resume_block = get_resume_block(course_outline_root_block) if course_outline_root_block else None
+        has_visited_course = bool(resume_block)
+        if resume_block:
+            resume_course_url = resume_block['lms_web_url']
+            resume_course_title = resume_block['display_name']
+        else:
+            resume_course_url = course_outline_root_block['lms_web_url'] if course_outline_root_block else None
+            resume_course_title = course_outline_root_block['display_name'] if course_outline_root_block else None
```

Comment:
I pulled in the `resume_course_title` field because it was wrapped up in the existing logic for these fields. It isn't used for the resume course button, but it's used for the next up banner.

Should I leave this field in for future use?

Reply:
The answer was to remove `resume_course_title` because we'll be removing "next up banner" logic. See: https://openedx.atlassian.net/browse/AA-256